docs/source/install.rst

```diff
@@ -65,102 +58,114 @@ According to the `contributors guidelines <contributors_guidelines>`__, you shou
     cd disdrodb
 
 
+
+
 Install the python developing environment
 ............................................
 
 You can use either conda or pip : 
 
-* **Conda (Option 1)**
-
-
-	* Create the `disdrodb-dev` conda environment:
-
-	.. code-block:: bash
-	
-		conda create --name disdrodb-dev python=3.9 --no-default-packages
-
-	* Activate the disdrodb conda environment:
 
-	.. code-block:: bash
+* **Pip**
 
-		conda activate disdrodb-dev
-	
-	* Check you are inside the disdrodb local repository, and install the required dependencies with:
-	
 	.. code-block:: bash
 
-		pip install -r requirements.txt
-	
-	.. warning::
-		Note: In future, when the conda disdrodb feedstock installation will be set up, the following code should be used instead: 
-	
-		.. code-block:: bash
-	
-	     		conda install --only-deps disdrodb
-		
-	* Install disdrodb 
+	   pip install -r requirements.txt
+	   
 	
-	.. code-block:: bash
+	We recomand you to use a virtual environment. This can be done as follow :
 	
-		pip install -e .
 	
-		
-* **Conda (Option 2)**
+	**With venv :**  
 
+		* Windows: Create a virtual environment with venv:
 
-	* Create the `disdrodb-dev` conda environment and install the required dependencies:
+			.. code-block:: bash
 
-	.. code-block:: bash
+			   python -m venv venv
+			   cd venv/Script
+			   activate
+			   pip install -r requirements.txt
+			   
 
-		conda env create -f environment.yml 
+		* Mac/Linux: Create a virtual environment with venv:
+
+			.. code-block:: bash
+
+			   virtualenv -p python3 venv
+			   source venv/bin/activate
+			   pip install -r requirements.txt
+
+	**With Conda env :**
 	
-	.. warning::
-		
-		Note: This command takes quite some time at the moment ... 
+		* Create the `disdrodb-dev` conda environment:
 
-	* Activate the disdrodb conda environment
+			.. code-block:: bash
+			
+				conda create --name disdrodb-dev python=3.9 --no-default-packages
 
-	.. code-block:: bash
+		* Activate the disdrodb conda environment:
+
+			.. code-block:: bash
 
-		conda activate disdrodb-dev
+				conda activate disdrodb-dev
 		
-	* Manually add the path of your local copy of disdrodb to the ``PYTHONPATH`` environment variable. 
-	  In Linux operating systems, you could add the following line to your ``.bashrc`` file located in the ``/home/<your_username>`` directory: 
-	  
-	  .. code-block:: bash
-	  	export PYTHONPATH="${PYTHONPATH}:/path/to/your/local/repo/of/disdrodb/"
+		* Check you are inside the disdrodb local repository, and install the required dependencies with:
+		
+			.. code-block:: bash
 
+				pip install -r requirements.txt
+			
+			.. warning::
+				Note: In future, when the conda disdrodb feedstock installation will be set up, the following code should be used instead: 
+			
+				.. code-block:: bash
+			
+					conda install --only-deps disdrodb
+			
+		* Install disdrodb 
+		
+			To install the project in editable mode : 
```

Comment:
This section should not be indented under conda env only. 
This applies to whatever environment (venv and conda). 

pip install -e is the way to install the package in dev/editable mode.
The alternative is to set the PYTHONPATH described here below.

Reply:
@ghiggi Let's keep that for the next PR if you dont mind